src/Command/CreateTrait.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal\Console\Command\ServicesTrait.
```

Comment:
`Drupal\Console\Command\ServicesTrait.`
must be
`Drupal\Console\Command\ CreateTrait.`